src/tapis-app/src/Sections/Streams/Instruments/Instruments.tsx

```diff
@@ -0,0 +1,58 @@
+import { OnSelectCallback } from 'tapis-ui/components/streams/instruments/InstrumentList';
+import { Config } from 'tapis-redux/types';
+import { Streams } from '@tapis/tapis-typescript';
+import { InstrumentList } from "tapis-ui/components/streams";
+import { InstrumentsListCallback } from 'tapis-redux/streams/instruments/types';
+import { default as VariablesAndMeasurements } from "../VariablesAndMeasurements";
+import { ListSectionList, ListSection, ListSectionBody, ListSectionHeader, ListSectionDetail, ListSectionDetailSection } from 'tapis-app/Sections/ListSection';
+import { Icon } from 'tapis-ui/_common';
+import { useInstruments } from 'tapis-redux';
+
+interface InstrumentsProps  {
+  project: Streams.Project,
+  site: Streams.Site,
+  config?: Config,
+  onList?: InstrumentsListCallback,
+  onSelect?: OnSelectCallback,
+  refresh?: () => void
```

Comment:
Other tapis-app sections do not have component properties because they are high level components that should not have clients. (onList, onSelect etc. would be tapis-ui component properties).